sdk/identity/identity/tests.yml

```
@@ -0,0 +1,12 @@
+trigger: none
+
+stages:
+  - template: /eng/pipelines/templates/stages/archetype-sdk-tests.yml
+    parameters:
+      PackageName: "@azure/identity"
+      ServiceDirectory: identity
+      TimeoutInMinutes: 20
+      EnvVars:
+        AZURE_CLIENT_ID: $(aad-azure-sdk-test-client-id)
+        AZURE_TENANT_ID: $(aad-azure-sdk-test-tenant-id)
+        AZURE_CLIENT_SECRET: $(aad-azure-sdk-test-client-secret)
```

We should avoid the use of these hardcoded variable group references. If you use the dynamic variable mappings below it's easier to add sovereign cloud testing support in the future.

```suggestion
        AZURE_CLIENT_ID: $(IDENTITY_CLIENT_ID)
        AZURE_TENANT_ID: $(IDENTITY_TENANT_ID)
        AZURE_CLIENT_SECRET: $(IDENTITY_CLIENT_SECRET)
        AZURE_SUBSCRIPTION_ID: $(IDENTITY_SUBSCRIPTION_ID)
```